tests/server/server-executeQuery_error-001.phpt

```diff
@@ -2,6 +2,7 @@
 MongoDB\Driver\Server::executeQuery() with invalid options
 --SKIPIF--
 <?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_clean(); ?>
```

Comment:
`skip_if_not_live()`. I believe this is the Server equivalent to `manager-executeQuery_error-002.phpt`.